app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryViewModel.kt

```diff
@@ -39,9 +39,9 @@ class PromotedStoryViewModel(
     )
   }
 
-  /** Starts [ResumeLessonActivity] if a saved exploration is selected
-   *  or [ExplorationActivity] if an un-started recommended story is selected.
-   */
+  /**
+   * Starts [ResumeLessonActivity] if a saved exploration is selected or [ExplorationActivity] if an
+   * un-started recommended story is selected.   */
```

Comment:
This will be addressed in a follow-up PR that's preparing for the 0.11 release. I'll add a link to the PR once it exists.

Marking this as resolved for the sake of getting the PR merged.

Reply:
This has been addressed in #5010.